desc/protoparse/parser.go

```diff
@@ -104,6 +104,18 @@ type Parser struct {
 	// with import statements all being relative to the root of this tree).
 	InferImportPaths bool
 
+	// DependencyDescriptors is a map of filenames to file descriptor protos,
+	// which will be consulted when resolving imports. This allows a compiled Go
+	// proto in another Go module to be referenced in the proto(s) being parsed.
+	//
+	// In the event of a filename collision, Accessor is consulted first,
+	// then FileDescriptors is consulted, and finally the well-known protos
+	// are used.
+	//
+	// In order to get descriptors from compiled Go protos, use the
+	// `desc.LoadFileDescriptor` method.
```

Comment:
nit: Go doc does not actually support markdown (sadly). Please remove the backticks.

Reply:
Done.